_health-care/_js/_components/mothers_maiden_name.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import _ from 'lodash';
+
+class MothersMaidenName extends React.Component {
+  constructor() {
+    super();
+    this.state = {name: ''};
+    this.handleNameChange = this.handleNameChange.bind(this);
+  }
+
+  handleNameChange(event) {
+    this.setState({name: event.target.value});
+  }
+
+  render() {
+    return (
+      <div>
+        <div className="usa-input-grid usa-input-grid-large">
+          <label htmlFor="veteran_mothers_maiden_name">Mother’s Maiden Name</label>
+          <input type="text" name="veteran[mothers_maiden_name]"
```

Comment:
Also I think the structure I'm suggesting is separate from whether the component is a question or not. It's more about keeping the component from having knowledge about the context it's being used in for proper information hiding. Otherwise you end up with more complicated unittests and more implicit requirements.